### DISCUSSION TITLE
Control volume level with mouse scroll

### DISCUSSION THREAD

**Discussion by jgbar** [f1multiviewer/issue-tracker#80] [Feature requests] [answered]
Request:  The ability to ajust the sound level (for all or just the active window) with the mouse scroll wheel and/or keboard shortcut.

**Comment by f1multiviewer** [comment, marked as answer]
Certainly possible, only issue is that some scroll wheels are way more sensitive than others (for instance, on macOS, I believe way more scroll events are sent)
